### ISSUE TITLE
OpenJDK SignedLoggerFinderTest 'CN=sample' missing from stdout/stderr 

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9#19825]
This is the 0.46 M2 build.

https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.openjdk_x86-64_linux_fips140_2_testList_1/47/
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.openjdk_x86-64_linux_fips140_2_testList_2/9/
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.openjdk_s390x_linux_fips140_2_testList_1/38
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.openjdk_s390x_linux_fips140_2_testList_2/6/
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.openjdk_ppc64le_linux_fips140_2_testList_2/7/
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.openjdk_ppc64le_linux_fips140_2_testList_1/38
jdk_lang_0, jdk_lang_1, jdk_lang_j9_0
java/lang/System/LoggerFinder/SignedLoggerFinderTest/SignedLoggerFinderTest.java
```
15:07:46  java.lang.RuntimeException: Unexpected fail.
15:07:46  	at SignedLoggerFinderTest.launchTest(SignedLoggerFinderTest.java:218)
15:07:46  	at SignedLoggerFinderTest.main(SignedLoggerFinderTest.java:102)
15:07:46  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:07:46  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
15:07:46  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:07:46  	at java.base/java.lang.reflect.Method.invoke(Method.java:575)
15:07:46  	at com.sun.javatest.regtest.agent.MainWrapper$MainTask.run(MainWrapper.java:138)
15:07:46  	at java.base/java.lang.Thread.run(Thread.java:857)
15:07:46  Caused by: java.lang.RuntimeException: 'CN=sample' missing from stdout/stderr 
15:07:46  
15:07:46  	at jdk.test.lib.process.OutputAnalyzer.shouldContain(OutputAnalyzer.java:221)
15:07:46  	at SignedLoggerFinderTest.launchTest(SignedLoggerFinderTest.java:214)
15:07:46  	... 7 more
```

**Comment by pshipton** [comment]
@jasonkatonica fyi

**Comment by pshipton** [comment]
@KostasTsiounis pls help find someone to take a look at this.

**Comment by KostasTsiounis** [comment]
> @KostasTsiounis pls help find someone to take a look at this.

Let me take a look. Is the error occurring only on 17?

**Comment by pshipton** [comment]
It's also occurring on jdk21.
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_sanity.openjdk_x86-64_linux_fips140_2/50/
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_sanity.openjdk_ppc64le_linux_fips140_2/34/
https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_sanity.openjdk_s390x_linux_fips140_2/34/

Testing for jdk11 hasn't run yet.

**Comment by pshipton** [comment]
It also fails on jdk11.
https://trssrtp1.fyre.ibm.com/allTestsInfo?buildId=668e74760a647c6331581285&limit=5&hasChildren=true
https://trssrtp1.fyre.ibm.com/allTestsInfo?buildId=668e74730a647c633158126f&limit=5&hasChildren=true

**Comment by KostasTsiounis** [comment]
The problem occurs from the fact that the JAR verification process specifies a distinct list of providers that can be utilized to verify signatures. Said list specifies providers using their simple name, but, after [this PR](https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/360), we only allow providers to be specified through their fully-qualified names when in `FIPS` mode.

After talking with @taoliult and @keithc-ca, I am working on a fix and I will open a PR, as soon as it is ready.

**Comment by keithc-ca** [comment]
All fixes have been merged and included in the 0.46.0 release branches.
